commit 404ffe5679e6430d37282d5eb258993cfda8dae4 (refs/changes/13/3213/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-01-15T19:19:19+00:00 (1 year, 9 months ago)
    
    TRITON-49 vms.changefeed.test.js throwing error makes whole VMAPI tests suite abort
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/test/vms.changefeed.test.js b/test/vms.changefeed.test.js
index dbf688c..179c6fc 100644
--- a/test/vms.changefeed.test.js
+++ b/test/vms.changefeed.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -97,7 +97,7 @@ function waitForValue(url, key, value, callback) {
             times++;
 
             if (times === TIMEOUT) {
-                throw new Error('Timeout waiting on ' + url);
+                callback(new Error('Timeout waiting on ' + url));
             } else {
                 setTimeout(function () {
                     waitForValue(url, key, value, callback);
